pkg/controller/networkpolicy/networkpolicy_controller_test.go

```diff
@@ -15,18 +15,186 @@
 package networkpolicy
 
 import (
+	"bytes"
 	"fmt"
 	"testing"
+	"time"
 
 	v1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/rand"
+	//"k8s.io/apimachinery/pkg/types"
```

Comment:
I think style-wise we should avoid commented-out imports?

Reply:
oops